nbs/core.ipynb

```diff
@@ -339,6 +339,8 @@
     "        self.id_col = id_col\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # Test if any of the inputs contains NaNs with available_mask = 1, fit shall raise error
</pre><p>Please add <code>#| polars</code> in the second line here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>